### PR TITLE
Deep Learning / Neural Networks

### PR DESCRIPTION
**Pull request by ChristosChristofidis**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -126,6 +126,7 @@
 - [Linguistics](https://github.com/theimpossibleastronaut/awesome-linguistics)
 - [Cryptography](https://github.com/MaciejCzyzewski/retter)
 - [Computer Vision](https://github.com/jbhuang0604/awesome-computer-vision)
+- [Deep Learning / Neural Networks](https://github.com/ChristosChristofidis/awesome-deep-learning)
```
Comment (sindresorhus) [inline readme.md]: `[Deep Learning](https://github.com/ChristosChristofidis/awesome-deep-learning) *(Neural networks)*`

**Comment by sindresorhus** [comment]
Noticed that your repo is missing a license and contributing.md. Would be nice to have. Feel free to steal the one in this repo ;)

**Comment by ChristosChristofidis** [comment]
Thanks for the suggestion. added :+1:

File: readme.md (hunk 1 above)
Comment (sindresorhus) [inline readme.md]: @ChristosChristofidis :arrow_up:

Comment (ChristosChristofidis) [inline readme.md]: Done

**Pull request closed without merging**